views.py

```diff
@@ -13,14 +13,13 @@ async def read_number_of_planets():
         return {"Number of Planets": planets["count"]}
 
 
-@app.post("/movie")
+@app.post("/movie/{movie_id}")
 async def get_movie_data(movie_id: int):
     response = requests.get(f"https://swapi.py4e.com/api/films/{movie_id}/")
```

Comment:
Please use only one lib for requests. Requests lib is not asynchronous. You already used httpx lib above. Please be consistent.

Reply:
Fixed